src/lib/message/L1ToL2Message.ts

```diff
@@ -382,27 +400,38 @@ export class L1ToL2MessageReader extends L1ToL2Message {
         fromBlock = toBlock
       }
     }
-    return null
+    return { redeemReceipt: null, expired: false }
   }
 
   /**
    * Has this message expired. Once expired the retryable ticket can no longer be redeemed.
+   * @deprecated Will be removed in v3.0.0
    * @returns
    */
   public async isExpired(): Promise<boolean> {
-    const currentTimestamp = BigNumber.from(
-      (await this.l2Provider.getBlock('latest')).timestamp
-    )
-    const timeoutTimestamp = await this.getTimeout()
+    return await this.retryableExists()
+  }
 
-    // timeoutTimestamp returns the timestamp at which the retryable ticket expires
-    // it can also return 0 if the ticket l2Tx does not exist
-    return currentTimestamp.gte(timeoutTimestamp)
+  private async retryableExists(): Promise<boolean> {
+    try {
+      const currentTimestamp = BigNumber.from(
+        (await this.l2Provider.getBlock('latest')).timestamp
+      )
+      const timeoutTimestamp = await this.getTimeout()
```

Comment:
```suggestion
      const timeoutTimestamp = await this.getTimeout()
      const currentTimestamp = BigNumber.from(
        (await this.l2Provider.getBlock('latest')).timestamp
      )
```
I think we can swap this 2 call because `getTimeout` might throw and we don't have to `getBlock`